swan-lake/development-tutorials/additional-tools/scan-tool.md

```diff
@@ -85,6 +85,98 @@ To ignore a specific set of rules during the analysis, use the following command
 $ bal scan --exclude-rules="ballerina:1, ballerina/io:2"
 ```
 
-## Publishing static code analysis reports to SonarQube.
+## Publish static code analysis reports to SonarQube.
 
-To learn how to publish reports to SonarQube, refer to [Configuration for Platform Plugins](https://github.com/ballerina-platform/static-code-analysis-tool/blob/main/docs/static-code-analysis-tool/ScanFileConfigurations.md#configuration-for-platform-plugins).
\ No newline at end of file
+SonarQube is a popular open-source platform for continuous inspection of code quality. It provides static code analysis, code coverage, and other features to help developers maintain clean, maintainable codebases. The Ballerina scan tool can be integrated with SonarQube to publish static code analysis reports, enabling seamless integration into your CI/CD pipelines.
+
+This guide walks you through the process of configuring SonarQube and publishing Ballerina static code analysis reports.
+
+### Prerequisites
+- SonarQube 9.9 LTA Community Edition installed.
+- SonarScanner CLI 4.8.0 or later installed, and added to your system `PATH`.
+- [SonarQube Ballerina plugin](https://github.com/ballerina-platform/sonar-ballerina/packages/2580146), and [SonarQube platform plugin](https://github.com/ballerina-platform/sonar-ballerina/packages/2580149) downloaded.
+
+### Configure the SonarQube server
+
+1. Install [Java 17](https://adoptium.net/en-gb/temurin/releases/?version=17) in your machine.
+   - If Java 17 is not the default Java installation, override it:
+
+      **For Unix/macOS:**
+      ```bash
```

Comment:
Remove the `bash` keyword.

Reply:
Fixed with 56a4413a009690234ac0707f58289ea3558b593e